### PR TITLE
update promotional banner

### PR DESCRIPTION
**Pull request by adamjkuhn**
### Summary
Update preheader promotional banner to highlight Kong Summit

### Reason
Ask from marketing direct via Slack.

### CODE REVIEW

File: app/_includes/nav-v2.html [hunk 1]
```diff
@@ -2,7 +2,9 @@
   <a class="skip-main" href="#main">Skip to content</a>
   <div id="promo-banner">
     <div class="container">
-      <div class="closebanner"></div> <strong><label>New</label> Konnect Plus: Pay-as-you-go from startup friendly to enterprise scale. <a target="_blank" href="https://konghq.com/pricing/">See Pricing &rarr;</a></strong></div>
+      <div class="closebanner"></div> <strong><label>EVENT</label> Kong Summit agenda announced! <a target="_blank"
+          href="https://konghq.com/kong-summit/#register">See Pricing &rarr;</a></strong>
```
Comment (lena-larionova) [inline app/_includes/nav-v2.html:6]: This link + link label don't really match. It says "See pricing", but the link takes you to a registration form with no mention of cost. Might it be better to either link to the top of the page (https://konghq.com/kong-summit), where you can see that it says "Free", or change the label to remove any mention of pricing?

**Review by falondarville — COMMENTED** [review]
Can you update the contrast? Perhaps we can darken the blue to reach WCAG AAA compliance.

**Comment by lena-larionova** [comment]
@adamjkuhn it looks like this banner is now being generated remotely and overwriting the banner in the docs site code - after talking to Dragos, it sounds like that's something you implemented recently. If this is the method that the product web team would rather take, then can we remove the files from this repo and manage them remotely only? At the moment, the existence of two versions of this banner on live pages is buggy:

1. The text is repeating itself:
Screen Shot 2021-09-02 at 2 24 17 PM

2. On page load, the original banner talking about webinars loads first, then it's transformed into the Summit banner.

**Review by lena-larionova — APPROVED** [review]
Approving to expedite the change.

**Comment by lena-larionova** [comment]
Merging, Dragos will disable the remote change to the banner.

**Pull request merged**